starters/http/dotnet/infra/core/monitor/applicationinsights.bicep

```diff
@@ -12,6 +12,7 @@ resource applicationInsights 'Microsoft.Insights/components@2020-02-02' = {
   properties: {
     Application_Type: 'web'
     WorkspaceResourceId: logAnalyticsWorkspaceId
+    DisableLocalAuth: true
```

Comment:
I think this needs to be a parameter and more generalized, so `main.bicep` could set it, and we dont hard code into a one-off core file.  When in doubt, a change to a core file means we need a more permanent fix, or a module to encapsulate our customization.  Can we please look at azure-dev repo and AVM for inspiration?  there might be a modern monitor definition that already exists and exposes this param.

Reply:
Good catch, fixed